README.md

```diff
@@ -1,2 +1,55 @@
 # drone-mavros
 File that runs onboard the drone, connects to and communicates with the ISAACS server
+
+## Setup
+### Simulation Installation Instructions
+Any repositories downloaded can be placed independently of the rest.
+1. Install the desired firmware SITL (Software-In-The-Loop). The two available options are PX4 or ArduPilot.
+    1. For ArduPilot, run the following: `git clone https://github.com/ArduPilot/ardupilot.git`
+    Change directory into ardupilot via `cd ardupilot`. Run `git submodule update --init --recursive`. 
+    From inside the ardupilot directory, run `Tools/environment_install/install-prereqs-ubuntu.sh -y`. Finally,
+    run `. ~/.profile`. For more information, visit https://ardupilot.org/dev/docs/building-setup-linux.html#building-setup-linux.
+    
+    2. For PX4, run the following: `git clone https://github.com/PX4/PX4-Autopilot.git --recursive`.
+    Change directory into PX4-Ardupilot via `cd PX4-Ardupilot`. Run `bash ./Tools/setup/ubuntu.sh`.
+    For more information, visit https://docs.px4.io/master/en/dev_setup/dev_env_linux_ubuntu.html.
+    
+2. Please install Gazebo by following [this link](http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install)
```

Comment:
Some ROS installations already have Gazebo installed. Not sure if this is worth mentioning.

Reply:
I just included it in case someone didn't have Gazebo.